playbooks/Mission_Control_Attribute_Lookup.yml

```diff
@@ -0,0 +1,19 @@
+name: MC Attribute Lookup
+id: d76dd338-b2a0-4916-abdc-40dff5924c70
+version: 1
+date: '2023-03-14'
+author: Kelby Shelton, Splunk
+type: Investigation
+description: Launches "attribute lookup" input playbooks, creates events, and then concludes the Response Tasks where this playbook appears.
+playbook: MC_Attribute_Lookup
```

Comment:
This needs to change to the following:
`Mission_Control_Attribute_Lookup`
Or else quite a few links will be broken.